CHANGELOG.md

```diff
@@ -8,6 +8,12 @@
 - Support for Azure Data Lake Storage Gen2 operations - List paths in Data Lake file systems via the command: `azmcp storage datalake file-system list-paths`. [#608](https://github.com/Azure/azure-mcp/pull/608)
 - Added new commands for Azure Function code generation and deployment best practices (https://github.com/Azure/azure-mcp/pull/630)
 - Add `azmcp sql firewall-rule list` command to list SQL server firewall rules. [[#610](https://github.com/Azure/azure-mcp/pull/610)]
+- Pre-requisites for onboarding cluster to Arc. [#650](https://github.com/Azure/azure-mcp/pull/650)
```

Comment:
Should we list the tools individually in the changelog?

Reply:
A single bullet describing the features and tools should be good I think.